yt_dlp/extractor/tubetugraz.py

```diff
@@ -45,7 +45,7 @@ def _extract_episode(self, episode_info):
             episode_info, ('mediapackage', 'creators', 'creator'), 'dcCreator', default='')))
         return {
             'id': id,
-            'title': title,
+            'title': title or id,
```

Comment:
If you cannot extract title, just leave it as None
```suggestion
            'title': title,
```

Reply:
if I leave it as None, the testcase fails because the "no title given, generic title used" warning seems to be an error in test mode

Reply:
add it to `expected_warnings` in the test

Reply:
ahh! did not know that field existed, thanks

Reply:
done